homeassistant/components/stream/__init__.py

```diff
@@ -124,6 +125,8 @@ def __init__(self, hass, source, options=None):
         if self.options is None:
             self.options = {}
 
+        self.ll_hls = self.options.pop(CONF_LL_HLS, False)
```

Comment:
We pop off the CONF_LL_HLS from the stream_options because the remaining options get passed along to pyav.